redbot/core/drivers/__init__.py

```diff
@@ -109,7 +117,7 @@ def get_driver(
         try:
             storage_type = BackendType(data_manager.storage_type())
         except RuntimeError:
-            storage_type = BackendType.JSON
+            storage_type = BackendType.MSGPACK
```

Comment:
Let's please not make this the default driver on same PR it is introduced

Reply:
as in discord, this isn't a default quite in that way. Completely understandable how that could happen, but we force the user to actually make a choice, and I'm unsure when this would ever hit this exception handling, but did not want to check if this one was safe to remove as part of this.